handlers/ecs/textextraction/app.py

```diff
@@ -0,0 +1,531 @@
+import os
+import io
+import re
+import json
+import uuid
+import tempfile
+import base64
+import logging
+import psycopg2
+import requests
+import boto3
+import sentry_sdk
+from enum import Enum
+from pathlib import Path
+from datetime import date, datetime
+from botocore.client import Config
+from botocore.exceptions import ClientError
+
+from fastapi import FastAPI
+from pydantic import BaseModel
+from typing import Union
+
+from deep_parser import TextFromFile, TextFromWeb
+from content_types import ExtractContentType, UrlTypes
+
+from s3handler import Storage
+
+logging.getLogger().setLevel(logging.INFO)
+
+SENTRY_DSN = os.environ.get("SENTRY_DSN")
+ENVIRONMENT = os.environ.get("ENVIRONMENT")
+sentry_sdk.init(SENTRY_DSN, environment=ENVIRONMENT, attach_stacktrace=True, traces_sample_rate=1.0)
+
+class InputStructure(BaseModel):
+    """Input Structure """
+    client_id: Union[str, None]
```

Comment:
Just for the information, `Union[str, None]` can have a shorthand `Optional[str]`.

Reply:
done.